lib/state/settings/reducer.ts

```diff
@@ -90,17 +90,21 @@ const noteDisplay: A.Reducer<T.ListDisplayMode> = (state = 'comfy', action) => {
 };
 
 const sendNotifications: A.Reducer<boolean> = (
-  state = Notification.permission === 'granted',
+  state = 'Notification' in window && Notification.permission === 'granted',
```

Comment:
should be possible to use optional chaining here, though maybe it has trouble given the top-level naming?

```ts
state = Notification?.permission === 'granted'
```

if it does complain then it should be a quick resolution, add `window`

```ts
state = window?.Notification?.permission === 'granted'
```

Reply:
I tried this `state = Notification?.permission === 'granted'` didn't work. Can't believe that I didn't try `window?.Notifications` my brain is not working today.

Reply:
that's an easy one. top-level bindings can be awkward. I think the difference is that property access is different than binding access. this is part of what makes JS ugly, because you can access things that don't exist or don't exist in the current scope.